lib/ex_webrtc/sdp_utils.ex

```diff
@@ -113,6 +114,7 @@ defmodule ExWebRTC.SDPUtils do
     |> Map.new()
   end
 
+  @spec get_payload_types(ExSDP.t()) :: %{non_neg_integer() => binary()}
```

Comment:
That's a little bit too generic. You still have to read the code to know what this function returns.

```suggestion
  @spec get_payload_types(ExSDP.t()) :: %{pt :: non_neg_integer() => mid :: binary()}
```